modules/javafx.controls/src/main/java/javafx/scene/chart/Axis.java

```diff
@@ -750,6 +748,9 @@ protected void tickMarksUpdated(){}
                 }
             }
             updateTickMarks(side, length);
+
+            // call tick marks updated to inform subclasses that we have updated tick marks
+            tickMarksUpdated();
```

Comment:
is this the only place where this call needs to be added?

for example, in BarChart the x axis is not updated after "Add series with duplicate category" - is this a separate issue?


![Screenshot 2024-06-25 at 11 58 52](https://github.com/openjdk/jfx/assets/107069028/672c0804-fd00-4bc1-9c01-bd89f41056dc)

Reply:
yes, the BarChart issue (which I actually want to get fixed) will need additional fixes in other classes. I could add those to this PR, but probably it's best to handle them in another ticket+PR to keep this PR affecting all `Chart`s minimal?

Reply:
a separate PR would be best.  I am getting lost - do we have a ticket for that issue already or do we need to create one?

Reply:
I think we need to create that as it turned out it's not the same as the minor tick problem fixed by this PR.

Reply:
but wait: this may not be a different issue.  I see a slightly different behavior relative to the master branch:

- select bubble chart page in the monkey tester
- add series with duplicate category

now I see the minor tick marks bunched up in the middle of Y Axis:

![Screenshot 2024-06-25 at 14 49 42](https://github.com/openjdk/jfx/assets/107069028/5d85367b-baee-4d4a-9450-3f7da0af49c5)

Reply:
ok, good, I did not manage to reproduce the minor tick problem on the Y axis in the bar chart unless I switch back to master.

Reply:
thank you for finding and fixing these issues!